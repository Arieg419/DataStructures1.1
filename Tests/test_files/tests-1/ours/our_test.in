Init 184
UpdateRottenFruits 7 1
UpdateRottenFruits 5 3
UpdateRottenFruits 3 1
UpdateRottenFruits 2 1
PlantTree 34 16
PlantTree 31 6
PlantTree 34 3
UpdateRottenFruits 7 2
UpdateRottenFruits 5 1
PlantTree 0 3
PlantTree 2 13
PlantTree 0 31
PlantTree 6 23
PlantTree 3 21
PlantTree 30 17
PlantTree 12 17
UpdateRottenFruits 5 2
Quit
