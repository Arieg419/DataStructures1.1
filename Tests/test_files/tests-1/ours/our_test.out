Init done.
UpdateRottenFruits: SUCCESS
UpdateRottenFruits: SUCCESS
UpdateRottenFruits: SUCCESS
UpdateRottenFruits: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
UpdateRottenFruits: SUCCESS
UpdateRottenFruits: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
PlantTree: SUCCESS
